server/services/forms/schemas/update-appeal-foi.js

```diff
@@ -0,0 +1,125 @@
+                .build()
+        ).withField(
+            Component('dropdown', 'IROfficerName')
+                .withValidator('required')
+                .withProp('label', 'Internal review officer name')
+                .withProp('choices', 'S_FOI_KIMU_TEAM_MEMBERS')
+                .withProp('visibilityConditions', [
+                    {
+                        'conditionPropertyName': 'appealType',
+                        'conditionPropertyValue': 'INTERNAL_REVIEW'
+                    }
+                ])
+                .withValidator('required', 'Internal review officer name is required')
+                .build()
+        )
+        .withField(
+            Component('radio', 'appealStatus')
+                .withProp('label', 'Has this been completed?')
+                .withProp('choices', [
+                    Choice('No', 'appealPending'),
+                    Choice('Yes', 'appealComplete')
+                ])
+                .build()
+        ).withField(
+            Component('date', 'dateSentRMS')
+                .withValidator('required')
+                .withValidator('isValidDate')
+                .withValidator('isBeforeToday')
+                .withProp('label', 'When was this completed?')
+                .withProp('visibilityConditions', [
+                    {
+                        'conditionPropertyName': 'appealStatus',
+                        'conditionPropertyValue': 'appealComplete'
+                    }
+                ])
+                .build()
+        ).withField(
+            Component('radio', 'appealOutcome')
+                .withValidator('required')
+                .withProp('label', 'What was the outcome?')
+                .withProp('choices', [
+                    Choice('Decision Upheld', 'DecisionUpheld'),
+                    Choice('Decision Part Upheld', 'DecisionPartUpheld'),
+                    Choice('Complaint Upheld', 'ComplaintUpheld')
+                ])
+                .withProp('visibilityConditions', [
+                    {
+                        'conditionPropertyName': 'appealStatus',
+                        'conditionPropertyValue': 'appealComplete'
+                    }
+                ])
+                .build()
+        ).withField(
+            Component('radio', 'complex')
+                .withValidator('required')
+                .withProp('label', 'Was the case complex?')
+                .withProp('choices', [
+                    Choice('Yes', 'complex-y'),
+                    Choice('No', 'complex-n')
+                ])
+                .withProp('visibilityConditions', [
+                    {
+                        'conditionPropertyName': 'appealStatus',
+                        'conditionPropertyValue': 'appealComplete'
+                    }
+                ])
+                .build()
+        ).withField(
+            Component('text-area', 'notes')
```

Comment:
like above, I've made a comment on the naming in the `hocs-data` repo.

Reply:
I have addressed this